CMakeLists.txt

```diff
@@ -8,10 +8,11 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release)
 endif()
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+include(GNUInstallDirs)
 
+# set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+# set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+# set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)

```

Comment:
I use this directories a lot during dev. We can toggle between GNUInstallDirs with an option e.g. USE_GNU_INSTALL_DIRS or PACKAGE_BUILD

Reply:
The usage of GnuInstallDirs is mandatory for the Package export, but does not affect the output directory settings used. So both could be used without any conflicts. Moved GNUInstallDirs to the package definition (src/CMakeLists.txt) and uncommented the old code